build-and-push-spring-boot-application/action.yaml

```diff
@@ -37,12 +37,13 @@ runs:
       shell: bash
       name: Generate the name of the docker image
       id: generate-image-name
-    - uses: gradle/actions/setup-gradle@v3
-      name: Build Docker Container
-      with:
-        arguments: |
-          ${{ inputs.gradle-module-name }}:clean
-          ${{ inputs.gradle-module-name }}:bootBuildImage
+    - name: Set Up Gradle
+      uses: gradle/actions/setup-gradle@v3
```

Comment:
Looks good to me. Is there a reason not to bump this to version 4 at the same time?

Reply:
Good question! My initial hesitation with bumping to setup-gradle@v4 was that the JJ's Web BE change to setup-gradle merely [bumped to @v3 and executed Gradle commands instead of passing `arguments`](https://github.com/detroit-labs/jjs-web-backend/pull/425), and that worked just fine. But since there are no changes required for the `with` inputs passed in to build-and-push-spring-boot-application or the environment (Node.js 20 and such), downstream workflows shouldn't be affected by the bump, I think. 🤔 

I'll push another commit that bumps setup-gradle to v4. If approved and merged, perhaps it's just a minor release for labs-cloud-actions/build-and-push-spring-boot-application and its kin.